packages/web-components/src/components/leaving-ibm/leaving-ibm-modal-body.ts

```diff
@@ -37,7 +37,10 @@ class C4DLeavingIbmModalBody extends StableSelectorMixin(CDSModalBody) {
     return html`
       <p><slot></slot></p>
```

Comment:
Should add a part here, too. Maybe something like:
```suggestion
      <p part="content"><slot></slot></p>
```